sections/main-product.liquid

```diff
@@ -220,19 +219,7 @@
                 <label class="quantity__label form__label" for="Quantity-{{ section.id }}">
                   {{ 'products.product.quantity.label' | t }}
                   <span class="quantity__rules-cart no-js-hidden{% if cart_qty == 0 %} hidden{% endif %}">
-                    <span class="loading-overlay hidden">
-                      <span class="loading-overlay__spinner">
-                        <svg
-                          aria-hidden="true"
-                          focusable="false"
-                          class="spinner"
-                          viewBox="0 0 66 66"
-                          xmlns="http://www.w3.org/2000/svg"
-                        >
-                          <circle class="path" fill="none" stroke-width="6" cx="33" cy="33" r="30"></circle>
-                        </svg>
-                      </span>
-                    </span>
+                    {%- render 'loading-spinner' -%}
```

Comment:
Interesting but this is happening in main too
https://screenshot.click/06-02-mqulf-5oouq.mp4

In your case it's not happening in main because the price block was disabled.

Reply:
Good catch! I agree- since it's not related to this PR let's create a separate issue